src/solvers/Makefile

```diff
@@ -164,13 +164,13 @@ SRC = $(BOOLEFORCE_SRC) \
       strings/format_specifier.cpp \
```

Comment:
:pick: The commit message is a bit misleading. "Doing this for all string functions" sounds a bit like this is done for all those functions in this commit, rather than planned for all of them for the future.